### ISSUE TITLE
Sui deploy smart contract error Failed to execute transaction TransactionDigest

### ISSUE THREAD

**Issue by Linhnef** [MystenLabs/sui#8618]
## Steps to Reproduce Issue

I try to deploy my smart contract, i successfully when deploy it in local but when i try to deploy it in DevNet it return error like that.
```
Failed to execute transaction TransactionDigest(EHUNaUZXRAwXui2Birh3z9uinmdF7ThzfKvPxfTzLwWu) with error RpcError(Call(Custom(ErrorObject { code: ServerError(-32000), message: "unexpected end of input", data: None })))
```
I can't find any documents to fix it.

Sorry if it's a noob question I'm a newbie.

**Comment by Mathman14** [comment]
experiencing same error

**Comment by Linhnef** [comment]
@Mathman14 I resolved it, that because my DevNet version is 0.25.0 I upgrade it to the lastest version and it work fine.
